media/css/plugincheck/plugincheck.less

```diff
@@ -263,8 +263,121 @@ html[dir="rtl"] {
 #sec-plugin-uptodate {
     display: none;
 }
-.footer-newsletter-form {
-    padding: 40px 20px;
+#mozorg-newsletter-form,
+#newsletter-form {
+    margin: 0 auto;
+    padding: 0 5px (@baseLine * 2);
+    background: transparent;
+    border-bottom: none;
+    box-shadow: none;
+    text-align: center;
+
+    h3 {
+        display: block;
+        float: none;
+        color: @textColorSecondary;
+        margin: 0;
+        padding: (@baseLine * 2) 0;
+        width: auto;
+        .font-size(2.4rem);
+        letter-spacing: -1px;
+
+        @media only screen and (max-width: @breakTablet) {
+            .font-size(2rem);
+        }
+    }
+
+    .form-contents {
+        float: none;
+        display: inline-block;
+        margin: 0;
+        width: 45%;
+        vertical-align: top;
+
+        @media only screen and (max-width: @breakTablet) {
+            display: block;
+            margin: 0 auto;
+            width: 100%;
+        }
+
+        input[type="email"] {
+            height: 32px;
+            padding: 5px 10px;
+            background: #fbf9f4;
+            border: 2px solid #e1ddd3;
+            border-radius: 7px;
+            -webkit-appearance: none;
```

Comment:
Copied from elsewhere in bedrock but yeah, let me add that. There is no reason I am aware of not to.